### PR TITLE
ITS: Improve CheckClusters.C, mv CheckTopologies.C

### PR DESCRIPTION
**Pull request by mconcas**

### CODE REVIEW

**Comment by mconcas** [comment]
@shahor02 @iouribelikov , just in case we want this upstream

File: Detectors/ITSMFT/ITS/macros/test/CreateDictionaries.C [hunk 1]
```diff
@@ -32,12 +32,12 @@
 #include <unordered_map>
 #endif
 
-void CheckTopologies(std::string clusfile = "o2clus_its.root",
-                     std::string hitfile = "o2sim_HitsITS.root",
-                     std::string collContextfile = "collisioncontext.root",
-                     std::string inputGeom = "",
-                     float checkOutliers = 2., // reject outliers (MC dX or dZ exceeds row/col span by a factor above the threshold)
-                     float minPtMC = 0.01)     // account only MC hits with pT above threshold
+void CreateDictionaries(std::string clusfile = "o2clus_its.root",
```
Comment (shahor02) [inline Detectors/ITSMFT/ITS/macros/test/CreateDictionaries.C]: could you add optional extraction of dX,dZ, and topology.hash() as discussed privately?

**Review by shahor02 — APPROVED** [review]

**Comment by mconcas** [comment]
@shahor02 as you approved this I guess it can be merged. Let me know If I need to add something.

**Comment by shahor02** [comment]
@mconcas did you compare the distributions of residuals from CreateDictionaries and CheckClusters with the dictionary?

**Pull request merged**